plugins/catalog/src/js/repositories/components/AddRepositoryFormModal.js

```diff
@@ -57,15 +58,16 @@ class AddRepositoryFormModal extends React.Component {
       {
         fieldType: "number",
         name: "priority",
-        placeholder: i18n._(t`Priority`),
+        placeholder: noRepositories ? "0" : i18n._(t`Priority`),
         required: false,
         min: "0",
         max: `${numberOfRepositories}`,
         step: "1",
+        disabled: noRepositories,
```

Comment:
Do we want to disable it completely or just the range part https://github.com/dcos/dcos-ui/pull/3444/files#diff-8e1f679d255276c6780d99e73aa03416R75?

Reply:
Good point.

Reply:
I think disabling it completely is okay, since we specify the value to be 0 in this case.